src/code/padmgr.c

```diff
@@ -44,132 +56,133 @@ void PadMgr_UnlockReleaseControllerQueue(PadMgr* padmgr, OSMesgQueue* ctrlrqueue
     }
 }
 
-void PadMgr_Lock2(PadMgr* padmgr) {
+void padmgr_LockContData(PadMgr* padmgr) {
     osRecvMesg(&padmgr->queue2, NULL, OS_MESG_BLOCK);
 }
 
-void PadMgr_Unlock2(PadMgr* padmgr) {
+void padmgr_UnlockContData(PadMgr* padmgr) {
     osSendMesg(&padmgr->queue2, NULL, OS_MESG_BLOCK);
 }
 
 #ifdef NON_MATCHING
 // regalloc and minor ordering differences
-void func_800C740C(PadMgr* padmgr) {
-    static u32 D_8012D284 = 0;
+void padmgr_RumbleControl(PadMgr* padmgr) {
+    static u32 errcnt = 0;
     static u32 D_8016A4F0;
     s32 temp;
-    s32 var1;
+    s32 tried_rumble_comm;
```

Comment:
youll want to make any snake_case variables camelCase to match out codebase style